pkgs/applications/science/biology/minc-tools/default.nix

```diff
@@ -14,15 +14,19 @@ stdenv.mkDerivation rec {
 
   patches = [ ./fix-netcdf-header.patch ];
 
+  postPatch = ''
+    # add missing CMake module to build NIFTI support
+    # (the maintainers normally build libminc and minc-tools in a meta-project)
```

Comment:
Let's put the comment outside the string body, so it's not set in the builder and doesn't cause rebuilds on updates.  Otherwise LGTM.

Reply:
Fixed, thanks.